src/common/schemas/Error.yml

```diff
@@ -3,14 +3,15 @@
       $ref: "./ErrorCode.yml"
     message:
       readOnly: true
-      description: message is a human-readable message.
+      description: Human-readable message.
       type: string
     op:
       readOnly: true
-      description: op describes the logical code operation during error. Useful for debugging.
+      description: Describes the logical code operation when the error occurred. Useful for debugging.
       type: string
     err:
       readOnly: true
-      description: err is a stack of errors that occurred during processing of the request. Useful for debugging.
+      description: Stack of errors that occurred during processing of the request. Useful for debugging.
       type: string
-  required: [code, message]
+  required:
+    - code
```

Comment:
@glinton @lesam Can you verify this change is OK? Just calling it out so I don't break a client.

Reply:
Thanks for the callout here! I'd be more concerned about the ui client generated from oats, as my guess is that is what drove those to be required fields.